manager/service/service.go

```diff
@@ -137,6 +137,16 @@ type Service interface {
 	DeleteSchedulerClusterToApplication(context.Context, uint, uint) error
 	AddSeedPeerClusterToApplication(context.Context, uint, uint) error
 	DeleteSeedPeerClusterToApplication(context.Context, uint, uint) error
+
+	GetModel(context.Context, types.ModelParams, types.ModelInfos) (*model.MachineModel, error)
+	GetModels(context.Context, types.ModelParams, types.ModelInfos) ([]*model.MachineModel, error)
+	UpdateModel(context.Context, types.ModelParams, types.ModelInfos) error
+	DeleteModel(context.Context, types.ModelParams, types.ModelInfos) error
+
+	GetVersion(context.Context, types.ModelParams, types.ModelInfos) (*model.Version, error)
```

Comment:
Add Version struct and VersionParams struct, it is not good to use the same struct as model.